website/settings.py

```diff
@@ -0,0 +1,121 @@
+"""
+Django settings for website project.
+
+Generated by 'django-admin startproject' using Django 2.0.7.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.0/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.0/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ')1jpnl5!2w@h)qytlp#960&y(s3wgr)5(n!iz&ku0q^box-r9%'

```

Comment:
Secret keys should not published, you need to recover it from environment,  we can discuss this later